SearchIn2DMatrix.java

```diff
@@ -0,0 +1,102 @@
+// Time Complexity : O(log(n))
```

Comment:
It's still O(log mn) (same as your other solution). Notice that your `high` is `m*n`.